### ISSUE TITLE
Request: option to use use local elm binary

### ISSUE THREAD

**Issue by urfolomeus** [madsflensted/elm-brunch#18]
This is more of a request than an issue. It occurred to me that seeing as you can now install the elm command line tool via npm, that you could install a project-specific version of elm and run from `node_modules/bin/elm` instead of `elm`. This would allow you specify elm as a project dependency and also lock to a particular version if need be.

With this being the case elm-brunch would need to be able to either take an option that would specify the  elm binary command to run, or that would check first for a local elm and default to global if a local not found.

What do you think? I don't mind adding this feature if you think it is worthwhile.

**Comment by madsflensted** [comment]
I guess you would normally have the elm binary location in your path of in the shell you run brunch in, but having an option to specify to location could maybe come in handy anyway.

**Comment by urfolomeus** [comment]
Cool, thanks. I'll have a look at it over the weekend. :)

**Comment by AdrianRibao** [comment]
+1 for this. Now I'm getting always

> /bin/sh: 1: elm: not found

and I have in bashrc:

`export PATH=./node_modules/.bin:$PATH`

So it could be great if we had an option for setting the executable to:

`$(npm bin)/elm`

**Comment by urfolomeus** [comment]
Yeah sorry, I've been tied up with other stuff so never got round to implementing. I _might_ have time over the coming week though :)
